src/components/MultiSelect/helpers/getAllValues.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { normalizeValue } from '.';
+
+export default function getAllValues(children) {
+    const values = [];
+    React.Children.forEach(children, child => {
```

Comment:
here there is an issue since this implies that options should be direct children of the Multiselect, if we do this:
```
<Multiselect>
    <div>
        <Option />
        <Option />
    </div>
</Multiselect>
```
then this will not work, can we use another approach?

Reply:
Done